client/wordpress-com.js

```diff
@@ -193,6 +193,16 @@ sections.push( {
 	secondary: true
 } );
 
+if ( config.isEnabled( 'account-recovery' ) ) {
+	sections.push( {
+		name: 'account-recovery',
+		paths: [ '/account-recovery' ],
+		module: 'account-recovery',
+		secondary: false,
+		enableLoggedOut: true,
```

Comment:
Any SSR'd pages can't include files that don't have the ssr-ready headers. To enable SSR for a particular route, I believe there's more wiring involved.